src/app.ts

```diff
@@ -195,21 +200,71 @@ export const createAppTask = withSpid({
     });
   });
   withSpidApp.post("/introspect", async (req, res) => {
```

Comment:
Does this API work only on _ non-jwt_ token?

Reply:
Nope it is used also to check if a token (JWT or not) has been previously blacklisted